scenes/game/local_game.gd

```diff
@@ -918,6 +919,7 @@ class Player:
 				deck.remove_at(0)
 				events += [parent.create_event(Enums.EventType.EventType_Draw, my_id, card.id)]
 			else:
+				parent.printlog("WARNING: probably shouldn't need to call reshuffle from draw anymore")
```

Comment:
What is this?

Reply:
oh I forgot to remove this; i was making sure nothing hit this part while testing